src/nft/components/details/detailsV2/InfoChips.tsx

```diff
@@ -0,0 +1,201 @@
+import { Trans } from '@lingui/macro'
+import { useWeb3React } from '@web3-react/core'
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { Unicon } from 'components/Unicon'
+import useENSAvatar from 'hooks/useENSAvatar'
+import useENSName from 'hooks/useENSName'
+import { useIsMobile } from 'nft/hooks'
+import { GenieAsset } from 'nft/types'
+import { getLinkForTrait } from 'nft/utils'
+import { ReactNode, useReducer } from 'react'
+import { ChevronDown, DollarSign } from 'react-feather'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ClickableStyle, EllipsisStyle, ExternalLink, LinkStyle, ThemedText } from 'theme'
+import { isAddress, shortenAddress } from 'utils'
+import { ExplorerDataType } from 'utils/getExplorerLink'
+import { getExplorerLink } from 'utils/getExplorerLink'
+
+const StyledBubble = styled(Row)`
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  padding: 10px 12px 10px 8px;
+  border-radius: 20px;
+  max-width: 144px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    max-width: 169px;
+  }
+`
+
+const StyledLabelMedium = styled.div`
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 20px;
+  color: ${({ theme }) => theme.textPrimary};
+
+  ${EllipsisStyle}
+`
+
+const StyledIcon = styled(Row)`
+  width: 24px;
+  height: 24px;
+  flex-shrink: 0;
+  color: ${({ theme }) => theme.accentAction};
+  border-radius: 100%;
+  overflow: hidden;
+  justify-content: center;
+  align-items: center;
+`
+
+const StyledLink = styled(Link)`
+  ${ClickableStyle}
+  ${LinkStyle}
+`
+
+const ConditionalLinkWrapper = ({
+  isExternal,
+  href,
+  children,
+}: {
+  isExternal?: boolean
+  href: string
+  children: ReactNode
+}) => {
+  return isExternal ? (
+    <ExternalLink href={href}>{children}</ExternalLink>
+  ) : (
+    <StyledLink to={href}>{children}</StyledLink>
+  )
+}
+
+const InfoBubble = ({
+  title,
+  info,
+  icon,
+  href,
+  isExternal,
+}: {
+  title: ReactNode
+  info: string
+  icon: ReactNode
+  href: string
+  isExternal?: boolean
+}) => {
+  return (
+    <ConditionalLinkWrapper isExternal={isExternal} href={href}>
+      <Column gap="sm">
+        <ThemedText.Caption color="textSecondary">{title}</ThemedText.Caption>
+        <StyledBubble gap="sm">
```

Comment:
nit: I think only the bubble should be clickable and have the hover state but fine either way
![Screen Shot 2023-05-15 at 09 00 47 ](https://github.com/Uniswap/interface/assets/11512321/35d1db29-8923-49bd-9687-4849ed632a78)

Reply:
good callout